testsuite/cli/src/client_proxy.rs

```diff
@@ -618,11 +623,18 @@ impl ClientProxy {
                 .get_txn_by_acc_seq(account, sequence_number - 1, true)
```

Comment:
I think you need to handle Network related issue here too. Otherwise one network issue and you will be done.

Reply:
I'm not sure what you mean. From what I can tell, the intent of this code is to just keep trying until it either completes or times out.